k4e-agent-sti.sh

```diff
@@ -70,6 +70,7 @@ fi
 # Install image-builder components
 #---------------------------------
 dnf install -y osbuild-composer composer-cli cockpit-composer bash-completion jq
+dnf install -y firewalld
```

Comment:
can we have this in the previous line?

Reply:
Sure

Reply:
what is the reason for installing firewalld?

Reply:
for 2 reasons: 
1. requirement of osbuilder, although we're not using cockpit for it.
2. for exposing the commit/iso/ks via the http-server

Reply:
If you do not install it then the HTTP server is not reachable?

Reply:
It wasn't reachable when I tested based on rhel-8.5 created by beaker.